librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -85,7 +85,7 @@
 #define ULT(x, y)   rz_il_op_new_ult(x, y)
 #define ULE(x, y)   rz_il_op_new_ule(x, y)
 #define UGT(x, y)   rz_il_op_new_ugt(x, y)
-#define UGR(x, y)   rz_il_op_new_ugr(x, y)
+#define UGE(x, y)   rz_il_op_new_uge(x, y)
```

Comment:
I suggest to send this change in a separate PR.

Reply:
Implemented in #2566.